api/libproject/devstart.rb

```diff
@@ -24,10 +24,10 @@ def add_default_options(parser, options)
   end
   parser.on("--account [ACCOUNT]",
        "Account to use when creating credentials (your.name@pmi-ops.org); "\
-       "use this or --creds-file") do |account|
```

Comment:
To clarify, it was a mistake to break the build, but moving from underscore to hyphen was intentional since hyphen word separators are more user-friendly in my opinion.